deploy.gradle

```diff
@@ -1,9 +1,23 @@
 apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+
+publishing {
+    publications {
+        SemverPublication(MavenPublication) {
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+            groupId rootProject.group
+            artifactId "semver"
+            version rootProject.version
```

Comment:
Aren't the last 3 fields: `groupId`, `artifactId` and `version` all implied from the rest of the build?

Reply:
You are right, I will remove it.